config/locales/en.yml

```diff
@@ -1,6 +1,78 @@
 # Sample localization file for English. Add more files in this directory for other locales.
 # See http://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
-en: 
+en:
```

Comment:
I would in the long term move the automated tests to use the same locale as we do.